### PR TITLE
Add table of contents for UG

### PR DESCRIPTION
**Pull request by Sherlock-YH**
Update PPP for Sherlock-YH

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,17 +1,40 @@
 # BankWithUs User Guide
 
 --------------------------------------------------------------------------------------------------------------------
-
-## Quick start guide
+## <span style="color:#CC0000">Table of contents:</span>
+* [Quick-start guide](#quick-start)
+* [Features](#features)
+    * [Help message](#help-message)
+    * [Add new account](#add-account)
```
Comment (vishnuvk47) [inline docs/UserGuide.md:8]: [Switch Account]
[Delete Account]
[View Account]
[Deposit]
[Withdraw]
[Transaction History]
[Save Goal]
[Withdraw Limit]
[Check Withdraw Limit]
[Delete Transaction History]
[Exiting the program]
[Frequently Asked Questions (FAQ)]

These links did not seem to work when I tried to click on them

**Review by vishnuvk47 — APPROVED** [review]
I think there is some broken links

**Review by vishnuvk47 — CHANGES_REQUESTED** [review]
There's some broken links

File: docs/UserGuide.md (hunk 1 above)
Comment (Sherlock-YH) [reply docs/UserGuide.md:8]: Now should be working

**Review by vishnuvk47 — APPROVED** [review]
LGTM!

**Pull request merged**